roles/scigateway/templates/apache.conf.j2

```diff
@@ -9,6 +9,7 @@
 <VirtualHost *:443>
   SSLEngine On
   SSLCertificateFile {{ scigateway_certificate_file }}
+  SSLCertificateChainFile {{ scigateway_certificate_bundle_file if scigateway_certificate_bundle_file is defined and scigateway_certificate_bundle_file|length else scigateway_certificate_file }}
```

Comment:
No need to check if scigateway_certificate_bundle_file is defined since it's defined in defaults